models/intermediate/latest_history/int_pendo__latest_nps_rating.sql

```diff
@@ -0,0 +1,47 @@
+with poll as (
+
+    select *
+    from {{ var('poll') }}
+
+    where lower(attribute_type) = 'npsrating'
+
+),
+
+poll_event as (
+
+    select *
+    from {{ var('poll_event') }}
+),
+
+limit_to_nps_polls as (
+
+    select 
+        poll_event.*
+    
+    from poll_event
+    join poll 
```

Comment:
For consistency can we specify the join

Reply:
what do ya mean?

Reply:
I assume you are intending this to be an `inner join`, could we specify that so it is consistent with our other naming conventions of joins.